scripts/deployment/trim_code_for_release.sh

```diff
@@ -19,4 +19,6 @@ rm -rf $REPOSITORY_FOLDER/vendor/squizlabs
 rm -rf $REPOSITORY_FOLDER/vendor/phpunit
 rm -rf $REPOSITORY_FOLDER/vendor/php-mock
 rm -rf $REPOSITORY_FOLDER/vendor/johnkary 
-rm -rf $REPOSITORY_FOLDER/scripts/
```

Comment:
`-rm -rf $REPOSITORY_FOLDER/scripts/` was already in line 16 so I deleted the duplicate one.